components/webui/README.md

```diff
@@ -0,0 +1,41 @@
+# Setup
+
+## Requirements
+
+* [NodeJS 14](https://nodejs.org/download/release/v14.21.3/)
+* [Meteor](https://docs.meteor.com/install.html#installation)
```

Comment:
It seems `docs/Building.md` only describes how to build the entire package using `task`.

I was thinking an even better approach is to add `meteor` installation into our Taskfile, so that users can just run the `task` command to get and build everything. However, it seems `meteor` does not support installation into a custom location, and the mitigation to override envvar `HOME` might introduce unexpected side effects. 

I agree we should have the users to install Meteor on their own.

Reply:
Good to know! Hopefully this is temporary anyway since we plan to replace Meteor.